src/main/java/com/vinylteam/vinyl/web/controller/AboutController.java

```diff
@@ -11,7 +11,7 @@
 
 @Controller
 @RequestMapping(path = "/about")
-@CrossOrigin(origins = {"http://localhost:3000", "http://react-wax-deals.herokuapp.com"})
+@CrossOrigin(origins = {"http://localhost:3000", "https://react-wax-deals.herokuapp.com"})
```

Comment:
if you have SecurityConfiguration.java this configure is unnecessary. I mean everywhere in project